src/vscode-bicep/package-lock.json

```diff
@@ -1,9547 +1,9 @@
 {
   "name": "vscode-bicep",
   "version": "0.0.0-placeholder",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
@bhsubra Can you update `npm` to the latest version and rerun `npm install`? We should use v2 lock file.

Reply:
Fixed as part of this commit:
https://github.com/Azure/bicep/pull/6060/commits/d196eac953c01620afdbe6f9b4da13b5fd30a98e

Thanks!